tests/ops/test_subset.py

```diff
@@ -615,7 +614,7 @@ def test_time_invariant_subset_with_time(load_esgf_test_data):
 
 
 # test known bug
-@pytest.mark.skipif(os.path.isdir("/badc") is False, reason="data not available")
+@pytest.mark.skipif(Path("/gws").is_dir() is False, reason="data not available")
```

Comment:
some of these have been changed to `/gws` but should be `/badc`

Reply:
Woops. Copy and paste has it drawback I suppose. Will fix!